### PR TITLE
DEVPROD-3884 Close cursor after accessing db

### PR DESCRIPTION
**Pull request by bynn**
DEVPROD-3884

### Description
cursor was never closed

### CODE REVIEW

**Review by ybrill — APPROVED** [review]
ty!

**Comment by ablack12** [comment]
LGTM -- there are two other instances of cursors we may want to close as well
https://github.com/evergreen-ci/evergreen/blob/5cc13e6c7e0f91d9be3b0723d0d95d2a0baa4a38/operations/admin_from_mdb.go#L247
https://github.com/evergreen-ci/evergreen/blob/7f5a91b97a07282bf61b7b9edf3ca4d2ee16e314/model/taskstats/db.go#L586

File: operations/admin_from_mdb.go [hunk 1]
```diff
@@ -270,7 +271,6 @@ func fromMdbForLocal() cli.Command {
 					return errors.Wrap(err, "writing buffer to tarball")
 				}
 
-				grip.Error(cursor.Close(ctx))
```
Comment (malikchaya2) [inline operations/admin_from_mdb.go:273]: Should this be moved to line 275? 
the lint task says: "operations/admin_from_mdb.go:244:5: for loop containing defer will not run until end of function"

File: model/event/host_event_finder.go [hunk 2]
```diff
@@ -50,22 +50,16 @@ func getRecentStatusesForHost(hostId string, n int) (int, []string) {
 		return 0, []string{}
 	}
 
-	out := []hostStatusDistro{}
-	for cursor.Next(ctx) {
-		doc := hostStatusDistro{}
-		if err := cursor.Decode(&doc); err != nil {
-			grip.Warning(err)
```
Comment (malikchaya2) [inline model/event/host_event_finder.go:57]: let's keep this grip.Warning

**Review by malikchaya2 — APPROVED** [review]
approved with nit

**Pull request merged**